query/src/servers/clickhouse/interactive_worker_base.rs

```diff
@@ -78,7 +78,7 @@ impl InteractiveWorkerBase {
         let ctx = session.create_query_context().await?;
         ctx.attach_query_str(query);
 
-        let (statements, _) = DfParser::parse_sql(query, ctx.get_current_session().get_type())?;
+        let statements = DfParser::parse_sql(query, ctx.get_current_session().get_type());
```

Comment:
How about returning a single `statement` directly?

So that we don't need to use `statements.0` anymore?

Reply:
mysql use the hint part, 
those will be removed recently soon，no bother to do the refact?

Reply:
LGTM